projects/ngx-seo/src/lib/seo-social-share/seo-social-share.service.ts

```diff
@@ -93,11 +93,16 @@ export class SeoSocialShareService {
     }
   }
 
-  public setImage(image?: string): void {
+  public setImage(image?: string, height?: number): void {
```

Comment:
could you make this more generic for width and height?

for example use this as the 2nd parameter:
```
dimensions?: {width?: number, height?:number}
```